SimTracker/TrackHistory/python/TrackQuality_cff.py

```diff
@@ -11,6 +11,8 @@
 		associatePixel = cms.bool(True),
 		associateStrip = cms.bool(True),
 		associateRecoTracks = cms.bool(True),
-		ROUList = copy.deepcopy(tabh.trackAssociatorByHits.ROUList)
+		ROUList = copy.deepcopy(tabh.trackAssociatorByHits.ROUList),
+                pixelSimLinkSrc = cms.InputTag("simSiPixelDigis"),
```

Comment:
@jpata, there is no need to carry around these other commits as part of this PR even if your newly added code functionally depends on them (actually, the TrackHistory fixes are totally decoupled from this PR). These other commits are already or will be integrated through separate PRs. So as long as these dependent PRs can be merged separately, I would keep them separate.